Library/Homebrew/cask/lib/hbc/container/cab.rb

```diff
@@ -8,9 +8,10 @@ class Cab < Base
       def self.me?(criteria)
         cabextract = which("cabextract")
 
+        # Ignores lines containing WARNING, but fails on other errors.
         criteria.magic_number(/^(MSCF|MZ)/n) &&
           !cabextract.nil? &&
-          criteria.command.run(cabextract, args: ["-t", "--", criteria.path.to_s]).stderr.empty?
+          criteria.command.run(cabextract, args: ["-t", "--", criteria.path.to_s]).stderr.split("\n").reject { |l| l.include? "WARNING" }.empty?
```

Comment:
> Just to make sure, is this an `stderr` check because errors can exit 0?

I really don't know, sorry.

Then again, I think we could remove this whole check, since it is also unlikely that the magic number matches, but it isn't a `cab` container. In the “worst” case, it will then fail when extracting it.